demo/config/targets/INST/lib/inst_protocol.rb

```diff
@@ -0,0 +1,25 @@
+# encoding: ascii-8bit
+
+# Copyright 2014 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+module Cosmos
+  module InstProtocol
```

Comment:
I don't think simulated target interface is going to support arbitrary protocols so this will probably be removed.